packages/odyssey-react/src/components/Icon/index.tsx

```diff
@@ -12,68 +12,91 @@
 
 // Code automatically generated by svgr; DO NOT EDIT.
 
-export * from "./Icon";
-export { default } from "./Icon";
```

Comment:
we discussed this previously but I am concerned about `<Icon />` being the default export as it contains all the SVG data in a single import. Naming this `IconDictionary` is part of my strategy to better clarify its intent and implementation but I'm open to other suggestions

Reply:
This one really falls under premature optimization for me. Having am `<Icon>` for folks to consume that can be refactored away if necessary seem like more than enough.